packages/website/components/messagebanner/messagebanner.js

```diff
@@ -24,16 +24,19 @@ export default function MessageBanner() {
 
   let maintenanceMessage = '';
 
+  const incidents = statusPageData?.incidents || [];
+
   const scheduledMaintenances =
     statusPageData?.scheduled_maintenances.filter(
       (/** @type {{ status: string; }} */ maintenance) => maintenance.status !== 'completed'
     ) || [];
-
   const { data: apiVersionData, error: apiVersionError } = useQuery('get-version', () => getVersion(), {
     enabled: (statusPageData && scheduledMaintenances.length === 0) || statusPageError !== null,
   });
 
-  if (scheduledMaintenances.length > 0) {
+  if (incidents.length > 0) {
+    maintenanceMessage = statusPageData?.incidents[0].incident_updates[0].body;
```

Comment:
What if perhaps `statusPageData?.incidents[0].incident_updates` expression evaluates to an empty array? Then I worry that the following `[0].body` could lead to a '.body does not exist on undefined' error at runtime.

What do you think of adding more optional chaining operators:
`statusPageData?.incidents?.[0].incident_updates?.[0]?.body`

```
> statusPageData = { incidents: [{ incident_updates: [] }] }
{ incidents: [ { incident_updates: [] } ] }
> statusPageData?.incidents[0].incident_updates[0].body
Uncaught TypeError: Cannot read property 'body' of undefined
> statusPageData?.incidents?.[0].incident_updates?.[0]?.body
undefined
> statusPageData = { incidents: [{ incident_updates: [{ body: 'foo' }] }] }
{ incidents: [ { incident_updates: [Array] } ] }
> statusPageData?.incidents?.[0].incident_updates?.[0]?.body
'foo'
```

Reply:
From what I had seen, if the incident exists, it always has a body, even if that body is an empty string. 

> What do you think of adding more optional chaining operators

_But_ we never know what might happen to an external source, so this makes sense to me and will help handle some exceptions, thanks!

Reply:
(also will add it to maintenance messages, which use a similar schema but are another object)

Reply:
nice ty.

also curious what you or others think about techniques like asserting the schema of the json response to get type safety. I think ajv could support thing like minimum array length assertions. https://ajv.js.org/